app/src/main/res/values/styles.xml

```diff
@@ -7,6 +7,7 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
         <item name="windowNoTitle">true</item>
+        <item name="android:textColor">#000000</item>
```

Comment:
I will be creating another another PR today to complete fragment classes. Will fix this and issues raised in that.